build/brave_channel.gni

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+declare_args() {
+  # Release channel support.
+  # "beta" for beta release.
+  # "dev" for dev channel release.
+  # "nightly" for nightly channel release.
+  # "" for stable channel release.
+  brave_channel = ""
```

Comment:
this is extracted to a separate file to allow it to be used in `.gni` files, otherwise there's a pretty bad import cycle that's hard to unwrap as we depend on it to make most of `sources.gni` available via `chrome_build.gni` and other global gnis.